src/node_util.cc

```diff
@@ -38,6 +38,11 @@ using v8::Value;
 #undef V
 
 
+static void IsPrimitive(const FunctionCallbackInfo<Value>& args) {
+  args.GetReturnValue().Set(!args[0]->IsObject() && !args[0]->IsExternal());
```

Comment:
Credit due to @trevnorris 

`%IsPrimitive()` isn't exposed anymore so as far as I understand this is the same.